packages/tools/devtools/devtools-core/src/test/DataVisualizerGraph.test.ts

```diff
@@ -205,4 +206,67 @@ describe("DataVisualizerGraph unit tests", () => {
 		};
 		expect(childCellTree).to.deep.equal(expectedChildCellTree);
 	});
+
+	it("Unknown object in Container Data", async () => {
+		const runtime = new MockFluidDataStoreRuntime();
+
+		const sharedCounter = new SharedCounter(
+			"test-counter-unknown-object",
+			runtime,
+			SharedCounter.getFactory().attributes,
+		);
+		sharedCounter.increment(42);
+
+		const sharedCell = new SharedCell(
+			"test-cell-unknown-object",
+			runtime,
+			SharedCell.getFactory().attributes,
+		);
+		sharedCell.set("Hello world");
+
+		const unknownObject = {};
+
+		const visualizer = new DataVisualizerGraph(
+			{
+				counter: sharedCounter,
```

Comment:
Once we're generating an unknown object node for the invalid data case (see my other comment), we can probably simplify this test quite a bit by _only_ specifying the invalid data here.

Reply:
I changed the unit test to only contain the `containerData` which is `unknown` to the visualizer. Also added an additional test case which passes an empty `containerData` to the `visualizer` for sanity check.